UC-Berkeley-Extension/global
# Removes the comment, history, and quickedit modules.

Removes three modules that we will not normally use.

To test:

* Checkout this branch
* Build the site if you have not already
  * From inside the VM, run `/var/www/html/scripts/setup.sh`
* Check that comment, history, and quickedit modules are disabled

* If you have already installed, from inside the VM run:
  * `/var/www/html/scripts/refresh.sh`
* Check that comment, history, and quickedit modules are disabled
